src/codecs/webp/lossless.rs

```diff
@@ -168,6 +168,30 @@ impl<R: Read> LosslessDecoder<R> {
         Ok(&self.frame)
     }
 
+    //used for alpha data in extended decoding
+    pub(crate) fn decode_frame_implicit_dims(
+        &mut self,
+        width: u16,
+        height: u16,
+    ) -> ImageResult<&LosslessFrame> {
+        let mut buf = Vec::new();
+        self.r.read_to_end(&mut buf)?;
+        self.bit_reader.init(buf);
+
+        self.frame.width = width;
+        self.frame.height = height;
+
+        let mut data = self.decode_image_stream(self.frame.width, self.frame.height, true)?;
+
+        for &trans_index in self.transform_order.iter().rev() {
```

Comment:
A note that each value in `transform_order` is 2-bit would be nice. It's local information in the other usage but non-obvious for this one, which makes the index below surprising on first sight.